hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -29,6 +30,8 @@
   // TODO (elingg) better handling in livestate and persistent state of zkfc task. Right now they are
   // chained.
   private Task zkfcNodeTask;
+  //Timed Health Check for node health monitoring
+  private Timer healthCheckTimer;
```

Comment:
yes, true, if we are going to keep it in the base. I will be pulling it out though.